echo_handlers.go

```diff
@@ -131,11 +131,12 @@ func (svc *Service) AppsListHandler(c echo.Context) error {
 	eventsCounts := make(map[uint]int64)
 	for _, app := range apps {
 		var lastEvent NostrEvent
-		var eventsCount int64
-		svc.db.Where("app_id = ?", app.ID).Order("id desc").Limit(1).Find(&lastEvent)
-		svc.db.Model(&NostrEvent{}).Where("app_id = ?", app.ID).Count(&eventsCount)
+		lastEventResult := svc.db.Where("app_id = ?", app.ID).Order("id desc").Limit(1).Find(&lastEvent)
 		lastEvents[app.ID] = lastEvent
-		eventsCounts[app.ID] = eventsCount
+		eventsCounts[app.ID] = 0
+		if lastEventResult.RowsAffected > 0 {
+			eventsCounts[app.ID] = 1
```

Comment:
what is that doing? 

that code looks strange to me and hard to read.

Reply:
it's a quick hack to remove a  query. But I think this is not the slow one, it's the other. I will close this, sorry